glpi-project/glpi
# Add Satitsfaction Surveys for Changes

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -

- Moved logic from TicketSatisfaction to new parent class CommonITILSatisfaction
- Added new notification templates for Change satisfaction surveys
- Added separate Entity configuration fields for Change surveys
- Added JS to hide the duration, max close date, and URL fields on the entity form when the survey rate is set to disabled

Still need to:
- Add new search options and test any existing ones related to surveys
- Add/expand tests (I need to figure out why I cannot mock the entity config values in tests)
- Update changelog

Functionally though, this feature should be mostly working.
